api/PayrollProcessor.Data.Persistence/Infrastructure/Clients/CosmosClientExtensions.cs

```diff
@@ -0,0 +1,33 @@
+using System.Linq;
+using PayrollProcessor.Data.Persistence.Features.Departments;
+using PayrollProcessor.Data.Persistence.Features.Employees;
+
+namespace Microsoft.Azure.Cosmos
+{
+    public static class CosmosClientExtensions
```

Comment:
These are some utilities to remove some boilerplate when working with the `CosmosClient`